docs/internals/security.txt

```diff
@@ -49,8 +49,13 @@ requires a security release:
 * The vulnerability is within a :ref:`supported version <security-support>` of
   Django.
 
-* The vulnerability applies to a production-grade Django application. This means
-  the following do not require a security release:
+* The vulnerability does not depend on manual actions performed by a site's
```

Comment:
I would not use "site" here, but rather "project". The reason being, a "site" for me is the running project, and thus the command might be run in production. Whereas "project" for me is more the code-base itself.

Say, there were an issue in the `createsuperuser` command which would leak the credentials somewhere. That'd be an issue since it's reasonable to expect the command to be run in production.

Reply:
Thank you, makes sense, will adjust.